packages/cli/src/commands/screenshot-diff/screenshot-diff.command.ts

```diff
@@ -36,24 +36,23 @@ export const diffScreenshots: (options: {
   headReplayId: string;
   baseScreenshot: PNG;
   headScreenshot: PNG;
-  threshold: number | null | undefined;
-  pixelThreshold: number | null | undefined;
+  diffOptions: ScreenshotDiffOptions;
   exitOnMismatch: boolean;
 }) => Promise<void> = async ({
   client,
   baseReplayId,
   headReplayId,
   baseScreenshot,
   headScreenshot,
-  threshold: threshold_,
-  pixelThreshold,
+  diffOptions,
   exitOnMismatch,
 }) => {
+  const { diffThreshold, diffPixelThreshold } = diffOptions;
   const logger = log.getLogger(METICULOUS_LOGGER_NAME);
 
-  const threshold = threshold_ || DEFAULT_MISMATCH_THRESHOLD;
-  const pixelmatchOptions: CompareImageOptions["pixelmatchOptions"] | null =
-    pixelThreshold ? { threshold: pixelThreshold } : null;
```

Comment:
Similar to above previously there was a bug here in that it was impossible to override pixelThreshold to 0 - setting it to 0 would set it to 0.1 (this is a danger of using falsey/truthy checks by default instead of `!= null` checks, and explicit `... != null && ... !== 0` checks when you do want to handle 0 and null case at same time)